templates/partial/_careers-available-roles.html

```diff
@@ -6,10 +6,10 @@ <h2 class="p-heading--4 js-filter__no-results u-hide">
   Sorry, there are no open positions matching your criteria.
 </h2>
 <div class="js-filter-jobs-container">
-  <ul class="p-list--divided js-job-list">
+  <ul class="p-list--accent-strip js-job-list">
```

Comment:
See the comment above, I don't think we need to create a new list variant, this could be a standard `p-list`.